osquery/filesystem/windows/fileops.cpp

```diff
@@ -871,22 +876,26 @@ ssize_t PlatformFile::write(const void* buf, size_t nbyte) {
             nret = -1;
           } else {
             // Error of unknown origin
+            TLOG << "Write to " << _fname << " failed with error code(" << GetLastError() << ")";
```

Comment:
Put a space between `code` and `(`, but also, you may be able to drop the "code" part. ".... with error (1234)" works.